code/modules/mob/living/simple_animal/hostile/monster.dm

```diff
@@ -48,4 +59,53 @@
 /obj/item/projectile/energy/neurotox
 	damage = 10
 	damage_type = TOX
-	icon_state = "toxin"
\ No newline at end of file
+	icon_state = "toxin"
+
+/mob/living/simple_animal/hostile/monster/cyber_horror
+	name = "cyber horror"
+	desc = "What was once a man, twisted and warped by machine."
+	icon_state = "cyber_horror"
+	icon_dead = "cyber_horror_dead"
+	icon_gib = "cyber_horror_dead"
+	speak = list("H@!#$$P M@!$#", "GHAA!@@#", "KR@!!N")
+	speak_emote = list("emits", "groans")
+	speak_chance = 15
+	turns_per_move = 5
+	see_in_dark = 6
+	maxHealth = 70
+	health = 70
+	melee_damage_lower = 5
+	melee_damage_upper = 15
+	attacktext = "flails around and hits"
+	move_to_delay = 5
+	can_butcher = 0
+	attack_sound = 'sound/weapons/hivehand_empty.ogg'
+
+/mob/living/simple_animal/hostile/monster/cyber_horror/Life(var/mob/living/simple_animal/hostile/monster/cyber_horror/M)
+	..()
+
+	if(health<maxHealth)
+		health=health+2                                                                        //Created by misuse of medical nanobots, so it heals
+		visible_message("<span class='warning'>[src]'s wounds heal slightly!</span>")
+
+/mob/living/simple_animal/hostile/monster/cyber_horror/emp_act(severity)
+	if(flags & INVULNERABLE)
+		return
+
+	switch (severity)
+		if (1)
+			adjustBruteLoss(40)
+
+		if (2)
+			adjustBruteLoss(20)
+
+/mob/living/simple_animal/hostile/monster/cyber_horror/Die()
+	..()
+	visible_message("<b>[src]</b> blows apart!")
+	var/obj/effect/decal/cleanable/blood/gibs/robot/R = getFromPool(/obj/effect/decal/cleanable/blood/gibs/robot, get_turf(src))
+	R.New(R.loc)
```

Comment:
I think I figured out what you meant. I took out all this bullshit in cyber_horror/Die() and hivebot/Die() and replaced it with a simple robot gibspawner